mmv1/third_party/terraform/services/gkehub2/data_source_google_gke_hub_membership_binding_test.go

```diff
@@ -0,0 +1,96 @@
+package gkehub2_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccDataSourceGoogleGKEHub2MembershipBinding_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+		"random_suffix":   acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckGKEHub2MembershipBindingDestroyProducer(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleGKEHub2MembershipBinding_basic(context),
+				Check: resource.ComposeTestCheckFunc(
+					acctest.CheckDataSourceStateMatchesResourceState("data.google_gke_hub_membership_binding.example", "google_gke_hub_membership_binding.example"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceGoogleGKEHub2MembershipBinding_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_project_service" "compute" {
+  service = "compute.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_project_service" "anthos" {
+  service = "anthos.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_project_service" "gkehub" {
+  service = "gkehub.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "time_sleep" "wait_for_gkehub_enablement" {
+  create_duration = "150s"
+  depends_on = [
+  google_project_service.gkehub, 
+  google_project_service.compute,
+  google_project_service.anthos
+  ]
+}
```

Comment:
I'd assume that we don't need service enablement for our testing project as all these services should've been already enabled. Can you try removing it and see if it still works? If so, we can save the sleep time for the test. Thanks.